tests/audiohelper.js

```diff
@@ -8,6 +8,10 @@ function getUserMedia() {
   return Promise.resolve({ id: 'default', getTracks: () => [] });
 }
 
+function beforeSetInputDevice() {
+  return Promise.resolve();
+}
```

Comment:
Can we remove all the default beforeSetInputDevice throughout this test file? We're already setting a default in the constructor so I think we don't have to pass this default.